content/notebooks/climate_indicators/PAVICStutorial_ClimateDataAnalysis-6Regridding&Conversion.ipynb

```diff
@@ -0,0 +1,1258 @@
+{
```

Comment:
<p>tabular</p><p> </p><p>data, please use ... instead of attempt ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/PAVICS-landing/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>